server/services/forms/index.js

```diff
@@ -547,10 +543,10 @@ module.exports = {
         }
     },
     getFormForCase: async (options) => {
```

Comment:
Would it be possible to have a `getFormForCase` for each specialised use? I think it might be easier to read if we had methods such as `getActionFormForCase` .etc

Reply:
After discussion decided to leave as is.